src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/esa/EsaIssueFinalDecisionAboutToSubmitHandlerTest.java

```diff
@@ -415,4 +412,39 @@ public void throwsExceptionIfItCannotHandleTheAppeal() {
         when(callback.getEvent()).thenReturn(EventType.APPEAL_RECEIVED);
         handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
     }
+
+    @Test
+    public void givenAnIssueFinalDecisionEvent_shouldUpdateStateAndDwpStateWhenStateIsNotReadyToListOrWithFta() {
+        commonSetUp();
```

Comment:
You should make this name more descriptive

Reply:
made names more descriptive

Reply:
I think it needs to be more descriptive than that really. E.g. setUpFinalDecisionNotice (if thats what its doing).